lib/cli/src/project_types.ts

```diff
@@ -258,9 +270,7 @@ export const supportedTemplates: TemplateConfiguration[] = [
 export const unsupportedTemplate: TemplateConfiguration = {
   preset: ProjectType.UNSUPPORTED,
   dependencies: {
-    // TODO(blaine): Remove when we support Vue 3
-    vue: (versionRange) => versionRange === 'next' || eqMajor(versionRange, 3),
-    // TODO(blaine): Remove when we support Vue 3
+    // TODO(blaine): Remove when we support Nuxt 3
     nuxt: (versionRange) => eqMajor(versionRange, 3),
```

Comment:
I specifically left nuxt here because they don't have v3 out yet and I want to make sure we work with it before allowing it in the generator.